### ISSUE TITLE
Hover target for Top Toolbar to slim in compact mode 

### ISSUE THREAD

**Issue by JaeTLDR** [zen-browser/desktop#942]
### What happened?

When hovering to reveal the Top Toolbar it is difficult and inconsistent to reveal the Top Toolbar, The side bar is much easier to trigger the reveal state. adding more size or a subtle hover landing zone could help.

### Reproducible?

- [ ] I have checked that this issue cannot be reproduced on Mozilla Firefox.

### Version

1.0.0-a.29

### Severity impact

Medium

### What platform are you seeing the problem on?

macOS - aarch64

**Comment by amtoaer** [comment]
Yes, I think both the top bar and the side bar are quite difficult to trigger. I just tested Arc browser and found that its side bar is easier to activate than Zen browser for two main reasons:
1. It has a wider activation zone and possibly a shorter activation time.
2. More importantly, once activated, it remains visible even if I move my cursor out of the window.

This design  allows me to simply slide my cursor out of the window slowly to activate the side bar, rather than having to find out the exact pixel to trigger the side bar and then hover over it.

**Comment by kittybwained** [comment]
On Windows, if Zen is in a non-maximized window and I'm in compact mode with "Hide the top toolbar as well in compact mode" turned on, I literally cannot get the top toolbar to show up no matter where I hover, it only works when Zen is maximized
